pkg/httphelper/client.go

```diff
@@ -456,3 +457,50 @@ func callNodeMgmtEndpoint(client *NodeMgmtClient, request nodeMgmtRequest, conte
 
 	return body, nil
 }
+
+func (client *NodeMgmtClient) CallIsFullQueryLogEnabledEndpoint(pod *corev1.Pod) (bool, error) {
+	client.Log.Info("client::callIsFullQueryLogEnabledEndpoint")
+	podHost, err := BuildPodHostFromPod(pod)
+	if err != nil {
+		return false, err
+	}
+	request := nodeMgmtRequest{
+		endpoint: "/api/v0/ops/node/fullquerylogging",
+		host:     podHost,
+		method:   http.MethodGet,
+		timeout:  time.Minute * 2,
+	}
+	isEnabledBytes, err := callNodeMgmtEndpoint(client, request, "")
```

Comment:
In Go, opt for short variable names. This "isEnabled" indicates it's a Java boolean, but that's not true at all. It could be "payload" for example to indicate what was fetched.

Reply:
Good idea, will implement as suggested. The response does indeed contain some additional fields about the MIME media type and stuff, so it really isn't a bool at all.